test/metabase/api/revision_test.clj

```diff
@@ -50,9 +50,9 @@
 
 ;; case with no revisions (maintains backwards compatibility with old installs before revisions)
 (deftest no-revisions-test
-  (testing "Loading revisions, where there are no revisions, should work"
+  (testing "Loading revisions, where there are no changes, should work"
     (t2.with-temp/with-temp [Card {:keys [id]}]
-      (is (= [{:user {}, :diff nil, :description nil, :has_multiple_changes false}]
+      (is (= [{:user {}, :diff nil, :description "modified this.", :has_multiple_changes false}]
```

Comment:
1. this is a test for backward compatibility when revision first introduced so I don't think it's relevant for new code because all new cards/dashboard will have a revision
2. since FE now use revision returned directly by BE, having `nil` is actually unacceptable, so I think this is a safe change.